contracts/Ramp.sol

```diff
@@ -70,23 +73,38 @@ contract Ramp is Verifier {
     mapping(uint256=>mapping(uint256=>bool)) public orderClaimedByVenmoId;
     mapping(uint256=>mapping(uint256=>OrderClaim)) public orderClaims;
 
+    mapping(bytes32 => bool) public nullified;
+
     /* ============ External Functions ============ */
 
-    constructor(uint256[rsaModulusChunksLen] memory _venmoMailserverKeys, IERC20 _usdc) {
+    constructor(uint256[rsaModulusChunksLen] memory _venmoMailserverKeys, IERC20 _usdc, uint256 _maxAmount) {
         venmoMailserverKeys = _venmoMailserverKeys;
         usdc = _usdc;
+        maxAmount = _maxAmount;
 
         orderNonce = 1;
     }
 
+    /* ============ Admin Functions ============ */
+
+    function setMaxAmount(uint256 _maxAmount) external onlyOwner {
+        maxAmount = _maxAmount;
+    }
+
+    function setVenmoMailserverKeys(uint256[rsaModulusChunksLen] memory _venmoMailserverKeys) external onlyOwner {
```

Comment:
Think it probably makes more sense to just store different keys instead of overwriting them? Then the user needs to specify which set of keys they want to use. Builds in a little more flexibility for us and catches the edge case where the keys are changed and there are valid emails using both keys